MangoPay/Refund.php

```diff
@@ -37,6 +37,12 @@ class Refund extends Transaction
      */
     public $RefundReason;
 
+    /**
+     * Message related to the refund
+     * @var string
+     */
+    public $RefundReasonMessage;
```

Comment:
Why did you add this param? I don't see it in the docs (correct me if I'm wrong)

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.